### PR TITLE
Create devcontainer.json

### PR DESCRIPTION
**Pull request by mishaDova**
# Add New Chain <chainId>

Thanks for your pull request to add a new support in Sourcify.

If you haven't done so, please follow the instructions on [how to request chain support](https://docs.sourcify.dev/docs/chain-support/) in docs.

Please check the following items before submitting your pull request for a speedy review.

## Checklist

- [ ] The branch is named as `add-chain-<chainId>`.
- [ ] I haven't modified the `chains.json` file directly.
- [ ] In `sourcify-chains.json` file
  - [ ] I've set `supported: true`.
  - [ ] I haven't added an `rpc` field but the one in [chains.json](../../src/chains.json) is used (if not, please explain why).
- [ ] I've added a test in [chain-tests.js](../../test/chains/chains-test.js) file.
- [ ] `test-new-chain` test in Circle CI is passing.

### CODE REVIEW

**Comment by kuzdogan** [comment]
Hi, thanks for the PR. Could you please elaborate a little the reason and use of this?

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (Chator1) [inline .devcontainer/devcontainer.json:1]: Abdulhafez

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+{
```
Comment (Chator1) [inline .devcontainer/devcontainer.json:2]: 0xFD689e5f2d8d9Aec0aD328225Ae62FdBDdb30328

File: .devcontainer/devcontainer.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
+  "features": {
+  }
```
Comment (Chator1) [inline .devcontainer/devcontainer.json:5]: 0xFD689e5f2d8d9Aec0aD328225Ae62FdBDdb30328

**Pull request closed without merging**